components/scream/src/share/io/tests/io.cpp

```diff
@@ -122,6 +122,7 @@ TEST_CASE("input_output_basic","io")
   auto avg_params = get_in_params("Average",io_comm);
   auto min_params = get_in_params("Min",io_comm);
   auto max_params = get_in_params("Max",io_comm);
+  auto multi_params = get_in_params("Multisnap",io_comm);
   Real tol = pow(10,-6);
```

Comment:
This seems like a very loose tolerance for double precision. Can we try to set `tol=100*std::numeric_limits<Real>::epsilon()`? It is already quite loose, given that no catastrophic cancellation should happen.